### PR TITLE
 RIA-9215 save case argument tab flag

### PR DESCRIPTION
**Pull request by ikirsanov**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RIA-9215

### Change description ###
Saving CASE_ARGUMENT_AVAILABLE with Yes value in ManageFeeUpdateHandler if it is DECISION_TYPE_CHANGED

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -1,51 +1,51 @@
 provider "azurerm" {
-    features {
-        resource_group {
-            prevent_deletion_if_contains_resources = false
-        }
+  features {
```
Comment (MiguelMartinRojas) [inline infrastructure/main.tf:2]: why this change?

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateHandlerTest.java [hunk 2]
```diff
@@ -1,174 +1,186 @@
...
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.CaseDetails;
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.CheckValues;
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
-import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.YesOrNo;
-import uk.gov.hmcts.reform.iacaseapi.domain.service.FeatureToggler;
-
-@MockitoSettings(strictness = Strictness.LENIENT)
```
Comment (MiguelMartinRojas) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateHandlerTest.java:37]: Why do you have changes for the whole file?

**Review by MiguelMartinRojas — CHANGES_REQUESTED** [review]

File: infrastructure/main.tf (hunk 1 above)
Comment (ikirsanov) [reply infrastructure/main.tf:2]: I think it is automatic commit

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateHandlerTest.java (hunk 2 above)
Comment (ikirsanov) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateHandlerTest.java:37]: nope locally it shows git blamed(other persons) and I as an author highligted only near my changes
